WordPress/UITestsFoundation/Screens/Editor/EditorPostSettings.swift

```diff
@@ -74,15 +73,15 @@ public class EditorPostSettings: BaseScreen {
             XCTAssertTrue(imageCount == 0, "Featured image is set but should not be")
         }
 
-        return EditorPostSettings()
+        return try EditorPostSettings()
     }
 
-    // returns void since return screen depends on which editor you're in
+    /// - Note: Returns `Void` because the return screen depends on which editor the user is in.
     public func closePostSettings() {
```

Comment:
We want the UITestsFoundation framework to be super easy to use for feature developers to write tests with. This little change in how the documentation is formatted means Xcode will show this information during autocompletion or if a user Alt-clicks on the method name.

<img width="511" alt="Screen Shot 2021-09-28 at 3 19 06 pm" src="https://user-images.githubusercontent.com/1218433/135027612-5c392151-f60d-4ace-b1bf-e2da5d20645e.png">